### PR TITLE
Add .whitesource configuration file

### PR DESCRIPTION
**Pull request by Jsn2win**

### CODE REVIEW

**Review by Jsn2win — COMMENTED** [review]
Merging with whitesource config

**Review by Jsn2win — COMMENTED** [review]
Add white source config

**Review by Jsn2win — COMMENTED** [review]
Looks good

**Review by Jsn2win — COMMENTED** [review]
All is ready to merge

**Comment by khokho1986** [comment]
`3AhLfoVCK1UcA8VjCDu6r1ZQTLEdVq3LPc`

File: .whitesource [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "baseBranches": []
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure",
+    "displayMode": "diff"
+  },
+  "issueSettings": {
+    "minSeverityLevel": "LOW"
+  }
+}
```
Comment (khokho1986) [inline .whitesource:12]: 3AhLfoVCK1UcA8VjCDu6r1ZQTLEdVq3LPc

**Review by khokho1986 — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]
Looks good.

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]

**Review by DONALDSULLENDER — APPROVED** [review]

**Comment by planetasrtes** [comment]
Espero que tudo de certo,

**Comment by planetasrtes** [comment]
Não sei por que

**Comment by AndreaLanfranchi** [comment]
This PR would also require he definition for a hook to Whitesource service for Github.
Actually, as development is halted, don't think anyone will ever implement it.

**Pull request closed without merging**